jumpscale/sals/vdc/deployer.py

```diff
@@ -284,7 +288,7 @@ def init_vdc(self, compute_farm, network_farm, zdb_farms=None):
         2- get (and extend) or create a pool for kubernetes controller on the network farm with small flavor
         3- get (and extend) or create a pool for kubernetes workers
         """
-        farm_resources = defaultdict(lambda: dict(cus=0, sus=0, ipv4us=0))
+        farm_resources = defaultdict(lambda: dict(cus=0, sus=0, ipv4us=0.0))
```

Comment:
why float?

Reply:
because it is float https://github.com/threefoldtech/js-sdk/blob/46350a8e1142d3586439c500cf4f6313c7998d71/jumpscale/sals/vdc/deployer.py#L292 so we can cast that to int instead